wishlist/style.css

```diff
@@ -0,0 +1,81 @@
+@import url("../../css/color.css");
+
+body {
+    font-family: Roboto, sans-serif;
+}
+
+.nav-header {
+    background-color: var(--primary);
+    color: var(--font-primary);
+    align-self: stretch;
+    padding: 0 5rem;
+}
+
+.menu-list-items {
+    padding: 1rem;
+}
+
+a {
+    color: var(--font-primary);
+    text-decoration: none;
+}
```

Comment:
Just a suggestion, if we can have `a` tag styling in base css so that it can be used everywhere.

Reply:
Sure, thanks 😄 
I will do the refactoring all together later